apps/tsconfig.json

```diff
@@ -23,7 +23,7 @@
     "jsx": "react",
     "target": "es5",
     "lib": ["ES2019", "DOM"],
-    "module": "es6",
+    "module": "es2020",
```

Comment:
TS won't let us use dynamic `import()` calls to replace dynamic `require()` calls unless we use `es2020` module syntax.
- Dynamic ESM `import()` calls, and `es2020` module syntax in general, should work on any browser except IE 10/11, which **currently has 1% browser market share** and is way past LTM.
- Its entirely possible that we should drop IE10/11 support and start targeting es2020 in general, dropping a lot of only-necessary-for IE10/11 pollyfills, build complexity, and outputting a slower transpiled format. This is a bigger discussion, but might be an easy one that gets a lot of wins.